pyproject.toml

```diff
@@ -1,3 +1,7 @@
+[tool.ruff]
+line-length = 88
+target-version = "py38"
```

Comment:
What's the lowest version still supported here?

Reply:
3.8 is correct. For worker minimum is 3.4.

Reply:
line-length should be 100.

Reply:
Done.